bootcamp/tutorials/integration/integrate_with_phidata.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
<p>Building Agent Workflows with Phidata and Milvus</p><p> </p><p>...</p><p> </p><p>Milvus vector database provides efficient storage and retrieval of any modality of data as embeddings. With Milvus and Phidata, you can easily integrate your knowledge into your Agent workflows. This document is a basic guide on how to use Milvus integration with Phidata.</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>